### ISSUE TITLE
[BUG] Golang syntax highlighting doesn't highlight anything

### ISSUE THREAD

**Issue by Owyn** [notepad-plus-plus/notepad-plus-plus#15538]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

I have just updated my n++ to the latest version cuz GO support was missing,
now I've gotten GO support (or so it claimed) but... it doesn't do anything

### Steps To Reproduce

1.  use 6.6.6 n+ version or so
2.  update to the latest
3.  see it fail to install new syntaxes

### Current Behavior

no highlighting

### Expected Behavior

yes highlighting

### Debug Information

```shell
Notepad++ v8.6.9   (64-bit)
Build time : Jul 12 2024 - 05:09:25
Path : S:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.4651
Current ANSI codepage : 1251
Plugins : 
    JSMinNPP (1.2202)
    Linter (0.1)
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

### Anything else?

![image](https://github.com/user-attachments/assets/878edc7f-5bea-40bf-bc22-6d8ef2f75ef9)
![image](https://github.com/user-attachments/assets/78a16934-71fb-4a29-b2dc-c1f3c9241e53)

while others work fine, e.g. action-script:
![image](https://github.com/user-attachments/assets/ef386bb1-30b0-498c-be88-4e49999c6fdf)

I have also tried Raku, Rust and Hollywood (ones which weren't there before I think) and they all do nothing, so it's the installer which failed I think

I ran the installer via admin and it didn't help

in the style configurator GO & Rust aren't even there at all:
![image](https://github.com/user-attachments/assets/3366eb95-9271-41af-8995-f993f13f1b7a)

**Comment by alankilborn** [comment]
Works for me in 8.6.9:

![image](https://github.com/user-attachments/assets/bdd28002-400b-4b9e-b533-8c87597344b9)

Probably your `stylers.xml` is out-of-date.

**Comment by Owyn** [comment]
> Probably your stylers.xml is out-of-date.

that's what I'm saying - updater fails to install new highlighters

**Comment by Owyn** [comment]
I have deleted old  stylers and langs files from my appdata and it created new ones which have go & rust but it didn't help anyhow, they are still missing from the style configurator in N++
![image](https://github.com/user-attachments/assets/966e1d79-db07-41ae-9b92-a072b386c9d2)

**Comment by Owyn** [comment]
Well, now my highlighting is trashed tho...
 all .go files now have no highlighting and I can't even set C++ one to work for .go extension by default cuz Go highlighter (which doesn't do anything) overrides it for some reason

is there a way to fix this? deleting appdata files didn't help

**Comment by chcg** [comment]
@Owyn Probably you need to update also the themes folder as you use the bespin theme.

**Comment by Owyn** [comment]
> @Owyn Probably you need to update also the themes folder as you use the bespin theme.

Thank you, deleting themes folder from the appdata worked and now I have highlighting
( I haven't changed anything in the theme's files before I'm pretty sure )

**Comment by pryrt** [comment]
@Owyn ,

For future use: I have just released a script for the PythonScript plugin which will help to automate incorporating new languages and styles from `stylers.model.xml` into your active `stylers.xml` and any themes you have on your computer (whether in %AppData% or the install directory).

The script, and how to use it, are described in [this Community Forum post](https://community.notepad-plus-plus.org/topic/26049/config-files-need-updating-too).  It was designed to _add_ the new stuff without deleting any of your customizations.

(This will even help your current installation: When I added Raku and Go syntax highlighting to Notepad++, I made sure to add those to all the themes, including your Bespin; however, other recently-added syntax highlighting languages, like Hollywood and GDScript and Rust, are still not included in Bespin or many of the other themes.  This script can fix even those discrepancies, so that Style Configurator will see _all_ the languages available in the `stylers.model.xml`.  But see the Community Forum post for a little about the color choices I had to make for the script-based updating to work.)
